test/app_web/controllers/capture_controller_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule AppWeb.CaptureControllerTest do
+  use AppWeb.ConnCase
+
+  @create_attrs %{text: "some text"}
+  @invalid_attrs %{text: nil}
+
+  describe "new capture" do
+    test "renders form", %{conn: conn} do
+      conn = get(conn, Routes.capture_path(conn, :new))
+      assert html_response(conn, 200) =~ "Capture"
```

Comment:
this test is fine for now. 
we will search for the placeholder text in a future iteration. 👍